misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZookeeperModule.kt

```diff
@@ -23,6 +23,7 @@ class ZookeeperModule(private val config: ZookeeperConfig) : KAbstractModule() {
     multibind<Service>().to(RepeatedTaskQueue::class.toKey(ForZkLease::class)).asSingleton()
     bind<LeaseManager>().to<ZkLeaseManager>()
     bind<CuratorFramework>().toProvider(CuratorFrameworkProvider::class.java).asSingleton()
+    install(ExecutorServiceModule.withFixedThreadPool(ForZkLease::class, "zk-lease-poller", 1))
```

Comment:
Can we make the `nThreads` configurable in `ZookeeperConfig`? 🤔

Reply:
I think clients won't really care about configuring this pool, since the client's usage shouldn't affect how many threads it needs for the zk lease poller task.